core/mo-services-impl/ccsds-com-impl/src/main/java/esa/mo/com/impl/archive/fast/FastObjectType.java

```diff
@@ -180,10 +180,10 @@ private static Long objectType2Mask(final ObjectType objType) {
         long versionVal = (objType.getVersion().getValue() == 0) ? (long) 0 : (long) 0xFF;
         long numberVal = (objType.getNumber().getValue() == 0) ? (long) 0 : (long) 0xFFFF;
 
-        return (new Long(areaVal << 48)
-                | new Long(serviceVal << 32)
-                | new Long(versionVal << 24)
-                | new Long(numberVal));
+        return (areaVal << 48
```

Comment:
Can someone double check that this doesn't change what it was doing before? Maybe @CesarCoelho since you wrote this. To me it looks good but with bit shifting you never now!

Reply:
I can't recall why I did an encapsulation with Long, maybe just noobing around with code...

Reply:
Should i change something?